movai_core_shared/core/zmq/zmq_client.py

```diff
@@ -87,12 +87,12 @@ def handle_socket_errors(self, exc: zmq.error.ZMQError, reset_socket=True) -> No
         if exc.errno == errno.ENOTSOCK:
             self._logger.warning(f"ZMQ socket error: {self._addr} got exception: {exc}.")
             if reset_socket:
-                self._logger.warning("Resetting ZMQ {self._addr} with potential data loss.")
+                self._logger.warning(f"Resetting ZMQ {self._addr} with potential data loss.")
```

Comment:
Maybe use leave interpolation to the logging function? https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html